source/lib-intro.tex

```diff
@@ -476,7 +476,7 @@
 appropriate):\footnote{To save space, items that do not apply to a function are omitted.
 For example, if a function does not specify any
 further
-preconditions, there will be no ``Requires'' paragraph.}
+preconditions, there will be no \requires paragraph.}
```

Comment:
I think section is best, because these can contain more than one paragraph, and my understanding is that clause and subclause have specific ISO meanings.

"element" is also used, but I think it sounds more vague than section.

But I agree that this can be done separately.